operations/helm/charts/mimir-distributed/Chart.lock

```diff
@@ -5,5 +5,8 @@ dependencies:
 - name: grafana-agent-operator
   repository: https://grafana.github.io/helm-charts
   version: 0.1.12
-digest: sha256:f57498f4a2d3db706e0739d38c877aac74a619a848be29b5b0ca7317ba339be2
-generated: "2022-08-17T11:05:13.740767-04:00"
+- name: rollout-operator
```

Comment:
I assume this checked in `.lock` file is intention.

Reply:
yes